clamscan/clamscan.c

```diff
@@ -295,6 +295,7 @@ void help(void)
     mprintf(LOGG_INFO, "    --phishing-sigs[=yes(*)/no]          Enable email signature-based phishing detection\n");
     mprintf(LOGG_INFO, "    --phishing-scan-urls[=yes(*)/no]     Enable URL signature-based phishing detection\n");
     mprintf(LOGG_INFO, "    --heuristic-alerts[=yes(*)/no]       Heuristic alerts\n");
+    mprintf(LOGG_INFO, "    --store-html-urls[=yes(*)/no]        Store html URLs in metadata\n");
```

Comment:
```suggestion
    mprintf(LOGG_INFO, "    --json-store-html-urls[=yes(*)/no]        Store html URLs in metadata\n");
```

What do you think about putting "json" as a prefix here so there's an indication it is related to the "gen-json" feature?

Reply:
Great idea!